apps/web/.env.development

```diff
@@ -33,12 +33,11 @@ API_HOST=http://localhost:3000
 
 # - - - - - - - - Auth - - - - - - - -
 
-AUTH_CLIENT_ID=<value goes here>
-AUTH_CLOUD_INSTANCE_ID=<value goes here>
-AUTH_TENANT_ID=<value goes here>
-AUTH_CLIENT_SECRET=<value goes here>
-AUTH_REDIRECT_URI=<value goes here>
-
-AUTH_INSPECTOR_GROUP_ID=<value goes here>
-AUTH_CASEOFFICER_GROUP_ID=<value goes here>
-AUTH_VALIDATIONOFFICER_GROUP_ID=<value goes here>
+AUTH_DISABLED=true
```

Comment:
Copy this one into the prod one and set it to false there.

Reply:
Done

Reply:
Done